tests/instancetype_test.go

```diff
@@ -1427,86 +1193,113 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 					},
 				},
 			}}
-			checkVMhasInferredInstancetypeAndPreference()
+			createAndValidateVirtualMachine()
 		},
 			Entry("and DataVolumeSourcePVC",
-				[]v1.DataVolumeTemplateSpec{{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: dataVolumeTemplateName,
-					},
-					Spec: cdiv1beta1.DataVolumeSpec{
-						Source: &cdiv1beta1.DataVolumeSource{
-							PVC: &cdiv1beta1.DataVolumeSourcePVC{
-								Namespace: util.NamespaceTestDefault,
-								Name:      pvcSourceName,
-							},
+				func() []v1.DataVolumeTemplateSpec {
+					dv := libdv.NewDataVolume(
+						libdv.WithNamespace(namespace),
+						libdv.WithForceBindAnnotation(),
+						libdv.WithPVCSource(sourceDV.Namespace, sourceDV.Name),
+						libdv.WithPVC(libdv.PVCWithAccessMode(k8sv1.ReadWriteOnce), libdv.PVCWithVolumeSize("1Gi")),
+					)
+					return []v1.DataVolumeTemplateSpec{{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: dataVolumeTemplateName,
 						},
-						Storage: &cdiv1beta1.StorageSpec{},
-					},
-				}}, nil,
+						Spec: dv.Spec,
+					}}
+				},
 			),
 			Entry(", DataVolumeSourceRef and DataSource",
```

Comment:
nit
```suggestion
			Entry("DataVolumeSourceRef and DataSource",
```

Reply:
This should result in a description of `should infer defaults from DataVolumeTemplates,  DataVolumeSourceRef and DataSource`?